game_for_class.py

```diff
@@ -8,70 +8,46 @@
 # Start with the player's class
 class Character:
     """All the methods the game player can use"""
-    def __init__(self,
-                 name: str,
-                 hp: float,
-                 attack: float,
-                 defense: float,
-                 speed: float,
-                 team: int):
+    def __init__(self, name: str, hp: float, attack: float, defense: float, speed: float, team: int):
+
         self.name = name
         self.hp = hp
         self.attack = attack
         self.defense = defense
         self.speed = speed
         self.team = team
+        """Initializes the Character"""
```

Comment:
Comments about the method go right under the arguments section. Also this message is stating the obvious